style.css

```diff
@@ -0,0 +1,473 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+}
+
+.text-color {
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 10%;
+  margin-left: 5%;
+  align-items: center;
+  padding: 6px;
+}
+
+.main-header {
+  background-color: #3c3a39;
+  width: 100vw;
+}
+
+.menubar {
+  height: 20px;
+}
+
+.section-1 {
+  background-color: #1c1a19;
+  height: 100vh;
+  width: 100vw;
+}
+
+.content-0 {
+  width: 100vw;
+  height: 100vh;
+  padding-top: 37%;
+  background-image: url(./socialmedia/design.png);
+  background-repeat: no-repeat;
+  background-position: 100% 100%;
+}
+
+.heading-1-container {
+  margin-bottom: 20px;
+}
+
+.margin-24 {
+  margin: 0 24px;
+}
+
+.heading-1 {
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+.heading-2 {
+  margin-left: 24px;
+  color: #fff5e1;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+.section-par {
+  color: white;
+  margin-left: 24px;
+  margin-top: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.social-media {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  list-style-type: none;
+  margin-top: 10%;
+  margin-left: 10%;
+  width: 50%;
+}
+
+/* my recent wor */
+
+.recent-profesional {
+  background: #e5e5e5;
+  margin-left: 6.4%;
+  margin-right: 6.4%;
+  width: 87.2%;
+}
+
+.gridd {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.my-recent-works {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  background-color: white;
+  flex: 1;
+  height: 100vh;
+  width: 100vw;
+  padding-bottom: 10%;
+}
+
+.H1 {
+  position: relative;
+  width: 100%;
+  margin-top: 15%;
+  margin-left: 10%;
+  font-size: 24px;
+  align-self: center;
+  color: #1c1a19;
+}
+
+.line {
+  height: auto;
+  display: flex;
+  position: relative;
+  margin-left: 10%;
+  margin-right: 10%;
+  width: 80%;
+  margin-top: 2.5%;
+  align-self: center;
+}
+
+.main-image {
+  display: flex;
+  margin-top: 15%;
+  width: 100%;
+  text-align: center;
+}
+
+.H2 {
+  position: relative;
+  width: 90%;
+  margin-top: 5%;
+  margin-left: 10%;
+  font-size: 18px;
+  color: #1c1a19;
+  order: 0;
+}
+
```

Comment:
- [ ] [REQUIRED] Please make changes! 
- 1-Please correct the position of the `line` under "My Recent Works" because it is not positioned correctly.
- 2-Change the color and font-type of the elements in this section just like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). 
- 3-Please adjust the "See Project" button as given design because its size is not correct.

![image](https://user-images.githubusercontent.com/63915024/170605755-ecc292a7-fdf0-486b-8b2b-3d384048f7f5.png)